cms/templates/asset_index.html

```diff
@@ -35,32 +35,31 @@
 
 <div class="wrapper-mast wrapper">
     <header class="mast has-actions has-subtitle">
-        <h1 class="page-header">
+        <h3 class="page-header">
             <small class="subtitle">${_("Content")}</small>
-            <span class="sr">&gt; </span>${_("Files & Uploads")}
-        </h1>
+            <span class="sr">- </span>${_("Files & Uploads")}
+        </h3>
 
-        <nav class="nav-actions" aria-label="${_('Page Actions')}">
-            <h3 class="sr">${_("Page Actions")}</h3>
+        <div class="nav-actions" aria-label="${_('Page Actions')}">
```

Comment:
@sarahkf I wasn't specific enough with my guidance here. We also want to remove the aria-label on this div. The presence of an aria-label will trigger an heuristic behavior that results in the screen reader treating this like an accessible object, which its not, and why we are converting it to a div.